docker/manage

```diff
@@ -148,19 +148,19 @@ build-solr() {
     'solr'
 }
 
-build-db() {
-  #
-  # db
-  #
-  echo -e "\nBuilding vcr-postgresql image ..."
-  ${S2I_EXE} build \
-    --copy \
-    -e "HTTP_PROXY=${HTTP_PROXY}" \
-    -e "HTTPS_PROXY=${HTTPS_PROXY}" \
-    '../server/db/config' \
-    'registry.access.redhat.com/rhscl/postgresql-10-rhel7:latest' \
-    'vcr-postgresql'
-}
+# build-db() {
+#   #
+#   # db
+#   #
+#   echo -e "\nBuilding vcr-postgresql image ..."
+#   ${S2I_EXE} build \
+#     --copy \
+#     -e "HTTP_PROXY=${HTTP_PROXY}" \
+#     -e "HTTPS_PROXY=${HTTPS_PROXY}" \
+#     '../server/db/config' \
```

Comment:
Are these configs getting applied to the new database?

Reply:
They aren't, no.